docs_en/tutorial/using-bind-mounts/index.md

```diff
@@ -67,7 +73,7 @@ So, let's do it!
 
     When you're done watching the logs, exit out by hitting `Ctrl`+`C`.
 
-1. Now, let's make a change to the app. In the `src/static/js/app.js` file, let's change the "Add Item" button to simply say
+1. Now, let's make a change to the app. In the `src/static/app.js` file, let's change the "Add Item" button to simply say
```

Comment:
The source is in `src/static/js/app.js` here too.

Reply:
![image](https://user-images.githubusercontent.com/2099300/67234113-236f9b80-f462-11e9-8fa5-b313227c4cbb.png)

Here's where app.js was when it came across in the zip.